src/main/kotlin/study/racinggame/domain/Race.kt

```diff
@@ -0,0 +1,28 @@
+package study.racinggame.domain
+
+import study.racinggame.service.RaceRule
+
+class Race(
+    private val carStorage: List<Car>,
+    private val raceRule: RaceRule
+) {
+
+    fun runTrack(): List<CarResponse> {
+        moveCarsIfPossible()
+        return carStorage.map(CarResponse.Companion::of)
```

Comment:
```suggestion
        return carStorage.map(CarResponse::of)
```

Companion이 빠져도 되겠죠~?